docs/rss-tutorials/path_planning_tutorial_preparations.md

```diff
@@ -11,7 +11,7 @@ To prepare for the Tutorial on Path Planning, only the [SMB core software](../co
 # Check software installation
 To test if the installation is working correctly, source the workspace where the softare is installed; then, the following need to be checked:
 
-1. The SMB Gazebo simulation is running properly (`$ roslaunch smb_gazebo sim.launch`)
+1. The SMB Gazebo simulation is running properly (`$ roslaunch smb_gazebo sim.launch launch_gazebo_gui:=true`)
```

Comment:
It might make sense to have a comment here, mentioning that you can show the gazebo gui by using the argument launch_gazebi_gui:=true